apiml-common/src/main/java/org/zowe/apiml/product/web/HttpConfig.java

```diff
@@ -88,13 +88,11 @@ public class HttpConfig {
     @Value("${server.maxTotalConnections:#{100}}")
     private Integer maxTotalConnections;
 
-    @Value("${apiml.httpclient.conn-pool.idleConnTimeoutSeconds:#{5}}")
+    @Value("${apiml.connection.idleConnectionTimeoutSeconds:#{5}}")
     private int idleConnTimeoutSeconds;
-    @Value("${apiml.httpclient.conn-pool.requestConnectionTimeout:#{10000}}")
+    @Value("${apiml.connection.timeout:#{10000}}")
```

Comment:
The default value here is different from the one in the start.sh. Shouldn't they be the same?

Reply:
And also it's different from ConnectionsConfig.java

Reply:
Yes, good catch. Bu the way, we have some redundancy because we defined another HttpConfig (`ConnectionsConfig.java`) for the new GW and we should clean it up at some point, use the config from apiml-common and choice the correct timeout values to use, but I'd rather do it as part of different PR/fix. Anyway I've changed the value on zaas start.sh to be consistent with the `apiml-common/src/main/java/org/zowe/apiml/product/web/HttpConfig.java` one.

Reply:
Well, I was talking about difference between gateway start.sh, ConnectionsConfig.java, NettyRoutingFilterApiml.java (60000) and HttpConfig.java (10000)

Reply:
I've raised the HttpConfig.java to 60000 too and added the todo to clean up/remove the ConnectionsConfig.java so we won't forget